app/api/speakers_calls.py

```diff
@@ -86,20 +86,29 @@ def before_get_object(self, view_kwargs):
         """
         if view_kwargs.get('event_identifier'):
             try:
-                event = self.session.query(Event).filter_by(identifier=view_kwargs['event_identifier']).one()
+                event = self.session.query(Event).filter_by(identifier=view_kwargs['event_identifier'],
+                                                            privacy='public').one()
```

Comment:
public and accepted

Reply:
Speaker calls are an invitation to submit a proposal for being a speaker at the event. They are either `private` or `public`. I didn't get the context of accepted here. Did you mean making changes for speakers to only return accepted speakers?